tests/conftest.py

```diff
@@ -28,3 +29,8 @@ def matplotlib_backend() -> None:
 @pytest.fixture(autouse=True)
 def torch_hub(tmp_path: Path) -> None:
     torch.hub.set_dir(tmp_path)
+
+
+@pytest.fixture(params=[True, False])
```

Comment:
This could probably be in `tests/models/conftest.py` since it's only relevant to those tests.